cdk/lib/security-vpc.ts

```diff
@@ -0,0 +1,11 @@
+import type { App } from '@aws-cdk/core';
+import type { GuStackProps } from '@guardian/cdk/lib/constructs/core';
+import { GuStack } from '@guardian/cdk/lib/constructs/core';
+import { GuVpc } from '@guardian/cdk/lib/constructs/vpc';
+
+export class SecurityVpc extends GuStack {
+  constructor(scope: App, id: string, props: GuStackProps) {
+    super(scope, id, props);
+    new GuVpc(this, 'SecurityVpc', { cidr: 'TODO - get from Enterprise.' });
```

Comment:
Would the CIDR be private or secret? If so we should draw this from https://github.com/guardian/private-infrastructure-config and [mock it for tests](https://github.com/guardian/private-infrastructure-config#using-with-jest) so it's not exposed in snapshots.

I'd suggest adding the VPC stack in a PR of it's own, to keep this change set in this PR small. Well, as small as it can be!

Reply:
I'm not sure why the CIDR would need to be private - do you know why this would matter? 

In any case, I should probably remove this file from this PR, as it's not in use yet. But will add it back in the follow up.